src/cswinrt/code_writers.h

```diff
@@ -2112,8 +2151,7 @@ Marshal.Release(inner);
                             {
                                 w.write("_defaultLazy = new Lazy<%>(() => (%)new SingleInterfaceOptimizedObject(typeof(%), _inner));", default_interface_name, default_interface_name, default_interface_name);
```

Comment:
We should also look at the scenarios where we use _defaultLazy as I think it might be similarly impacted like _lazyInterfaces where the internal constructor was the one which sets it.

Reply:
Addressed by adding this back to the public constructor for the same edge case